src/index.html

```diff
@@ -6,7 +6,316 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo-link">
+          <img
+            class="header__logo-img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="header__nav">
+          <ul class="header__nav-list">
+            <li class="header__nav-list-item">
```

Comment:
```suggestion
            <li class="nav__item">
```

Reply:
Not needed to inherit name of element, it should be based on block. In example i've proposed it's `nav` 
so element is `nav__item`